packages/server/src/api/controllers/static/index.ts

```diff
@@ -165,12 +225,27 @@ export const serveApp = async function (ctx: Ctx) {
   try {
     db = context.getAppDB({ skip_setup: true })
     const appInfo = await db.get<any>(DocumentType.APP_METADATA)
+
     let appId = context.getAppId()
+    const manifest = await getComponentLibraryManifest(
```

Comment:
I'm not sure about needing to do this everytime, it will likely slow down app loading time a fair bit, as we need to pull these files from S3 (with varied performance).

Is it possible that we could use the app version, rather than a flag in the manifest for this? Or could we update something into the app metadata to state that the skeleton loader can be used?

If not, perhaps there is a way we can cache this, but it would be good if we could note this somewhere in the app metadata rather than needing to pull the manifest as it will be a lot faster.